Display.cpp

```diff
@@ -288,20 +300,44 @@ void Display::update(const tm *now, bool showWeather)
     sprintf(buffer, "%02d/%02d", now->tm_mday, daysInMonth);
     _display->drawText(buffer, FONT_MEDIUM, LEFT, 394);
 
-    #ifdef SHOW_DAY
-    // Day name
-    _display->drawText(I18N_DAYS[now->tm_wday], FONT_MEDIUM, RIGHT, 394, DisplayGDEW075T7::TOP_RIGHT);
+    #if defined(SHOW_DAY) || defined(BIRTHDAYS)
+      bool showDay = true;
+      #if defined(BIRTHDAYS) && defined(SHOW_NAME)
+        if (activeBirthday != nullptr && !showWeather){
+          showDay = false;
+          _display->drawText(activeBirthday->name, FONT_MEDIUM, RIGHT, 394, DisplayGDEW075T7::TOP_RIGHT);
+        }
+      #endif  
+      #ifdef SHOW_DAY
+        if(showDay){
+          // Year
+          _display->drawText(I18N_DAYS[now->tm_wday], FONT_MEDIUM, RIGHT, 394, DisplayGDEW075T7::TOP_RIGHT);
+        }
+      #endif
     #endif
```

Comment:
Hey! Sorry I didn't review this earlier, I kinda took a break from this project. I don't really like the idea of replacing the day with this information because if they're doing a first + last name that could pretty easily overflow the amount of space available. Maybe this could be underneath the chamber icons using FONT_SMALL, and combined with the age in a format like "{name}, {age}"?

Reply:
That would also remove the need for the name to be in all caps, since FONT_SMALL contains the full set of uppercase and lowercase characters.